extensions/dacpac/src/wizard/pages/dacFxSummaryPage.ts

```diff
@@ -10,22 +10,12 @@ import { DataTierApplicationWizard, Operation } from '../dataTierApplicationWiza
 import { BasePage } from '../api/basePage';
 
 export class DacFxSummaryPage extends BasePage {
-
-	protected readonly wizardPage: azdata.window.WizardPage;
-	protected readonly instance: DataTierApplicationWizard;
-	protected readonly model: DacFxDataModel;
-	protected readonly view: azdata.ModelView;
-
 	private form: azdata.FormContainer;
 	private table: azdata.TableComponent;
 	private loader: azdata.LoadingComponent;
 
 	public constructor(instance: DataTierApplicationWizard, wizardPage: azdata.window.WizardPage, model: DacFxDataModel, view: azdata.ModelView) {
-		super();
-		this.instance = instance;
-		this.wizardPage = wizardPage;
-		this.model = model;
-		this.view = view;
+		super(instance, wizardPage, model, view);
```

Comment:
@kisantia do you remember if there was any particular reason as to why they were added here and the base class? just checkin so that we dont regress unintentionally.

Reply:
I asked Kim when I first noticed this; she also didn't know why it was this way/if it was done intentionally.

Reply:
yeah this was done unintentionally when I was just trying to make something work in ADS for the first time. It was a pretty big PR which is why it didn't get caught back then.